app/scripts/controllers/transactionsAddReqResModal.js

```diff
@@ -7,15 +8,35 @@ angular.module('openhimWebui2App')
     $scope.viewFullBody = false;
     $scope.viewFullBodyType = null;
     $scope.viewFullBodyContent = null;
+    $scope.fullBodyTransformLang = null;
+    
+    // transform request body with indentation/formatting
+    if( record.request && record.request.body ){
+      if( record.request.headers && record.request.headers['content-type'] ){
```

Comment:
Do you know if this check is case insensitive? Else we need to check, because it could come through as `Content-Type`